docs/configuring.asciidoc

```diff
@@ -18,10 +18,11 @@ The following topics describe how to configure APM Server:
 * <<configuration-path>>
 --
 
-:only-elasticsearch:
```

Comment:
I guess this is not easy, but would be nice to mention that the APM UI only works with elasticsearch output...

Reply:
In `Load the Elasticsearch index template` we will have a `load the template manually` section for users of non-elasticsearch outputs.